.github/actions/get-changelog/action.yml

```diff
@@ -5,4 +5,4 @@ outputs:
     description: 'The current changelog'
 runs:
   using: 'node12'
-  main: 'index.js'
+  main: 'transpiled/.github/actions/get-changelog/index.js'
```

Comment:
Also, it would be great if we wouldn't transpile .d.ts files

Reply:
I removed the .d.ts files. However I can not get lesser deep folder structure. If I put the root dir to the action, it is es desired from the folder structure but the compiler complains because not all source files are under root. The `rootDirs` make the import easier but they do not change the generated result. What would work is making a proper module out of the script files folder, but this seems a bit too much effort.

Reply:
I agree. Still, consider transpiling it to `dist`.